src/program/lwaftr/run/run.lua

```diff
@@ -53,6 +53,21 @@ function parse_args(args)
          fatal(("Couldn't locate configuration file at %s"):format(conf_file))
       end
    end
+   function handlers.cpu(arg)
+      local cpu = tonumber(arg)
+      if not cpu or cpu ~= math.floor(cpu) or cpu < 0 then
+         fatal("Invalid cpu number: "..arg)
+      end
+      local cpu_set = S.sched_getaffinity()
+      cpu_set:zero()
+      cpu_set:set(cpu)
+      S.sched_setaffinity(0, cpu_set)
+   end
+   handlers['real-time'] = function(arg)
+      if not S.sched_setscheduler(0, "fifo", 1) then
+         fatal('Failed to enable real-time scheduling.  Try running as root.')
```

Comment:
There are ways to configure snabb to be launched by a user without root perms.  OTOH it's important to check the return values here, and the only way the call can fail is if you're not root, so the message makes sense to me.